app/styles/ember-arcgis-portal-components.scss

```diff
@@ -209,11 +211,8 @@ div.panel-body:has(.item-picker) {
       margin: 10px 0px;
     }
     .item-picker-current-item-preview {
-      position: absolute;
-      height: calc(100% - 10px);
-      width: calc(100% - 30px);
-      top: 1em;
-      bottom: 10%;
+      display: flex;
+      flex-direction: column;
       overflow: auto;
       min-height: 400px;
```

Comment:
I don't think we need this rule anymore. Also, the min-height: 45rem above.

Reply:
Instead, I think we we would want something like:
```
height: 100%;
justify-content: space-between;
```

To make that space between keep everything near the top but push the buttons to the bottom, you would need to wrap the `h4`, `.item-picker-current-item-preview-meta`, `.item-picker-current-item-preview-description`, & `.item-picker-current-item-preview-meta` in a `div`.